go/libraries/doltcore/table/untyped/csv/writer.go

```diff
@@ -240,13 +247,13 @@ func (csvw *CSVWriter) write(record []*string) error {
 // 		of Microsoft Excel and Google Drive.
 // 		For Postgres, quote the data terminating string `\.`.
 //
-func (csvw *CSVWriter) fieldNeedsQuotes(field *string) bool {
+func fieldNeedsQuotes(field *string) bool {
 	if field != nil && *field == "" {
 		// special Dolt logic
 		return true
 	}
 
-	if *field == `\.` || strings.Contains(*field, csvw.info.Delim) || strings.ContainsAny(*field, "\"\r\n") {
+	if *field == `\.` || strings.Contains(*field, ",") || strings.ContainsAny(*field, "\"\r\n") {
```

Comment:
This seems wrong. There are other delimiters possible.

Reply:
Should have a test

Reply:
Adressed